src/read.c

```diff
@@ -3541,6 +3541,10 @@ static avifResult avifParseMovieBox(avifDecoderData * data,
 
         AVIF_CHECKERR(avifROStreamSkip(&s, header.size), AVIF_RESULT_BMFF_PARSE_FAILED);
     }
+    if (data->tracks.count == 0) {
+        avifDiagnosticsPrintf(data->diag, "moov box did not contain any tracks");
```

Comment:
Nit/optional: We can use the present tense "does" instead of the past tense "did".

Reply:
Done.